CS.Security/Controllers/UsersController.cs

```diff
@@ -12,7 +12,7 @@ namespace CS.Security.Controllers
     public class UsersController : ControllerBase
     {
         private readonly IUserService _userService;
-        private readonly string url = Environments.FrontAddress;
+        private readonly string _url = Environments.FrontAddress;
```

Comment:
Using ReSharper/Rider could help you solve the issues like that, it will actually give you hints every time when code convention is violated. Readonly properties should start from underscore.